extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -327,8 +412,16 @@ export class ProjectsController {
 		}
 
 		if (success) {
+			TelemetryReporter.createActionEvent(TelemetryViews.ProjectTree, 'deleteObjectFromProject')
+				.withAdditionalProperties({ objectType: node.constructor.name })
+				.send();
+
 			this.refreshProjectsTree(context);
 		} else {
+			TelemetryReporter.createErrorEvent(TelemetryViews.ProjectTree, 'deleteObjectFromProjectFailed')
```

Comment:
there's no additional data beyond the view name.  Should success vs. failure be a property and both cases share the same action name?

Reply:
Hm, well maybe at least add in what type of object you were trying to delete? But yeah if you don't really care about that then this is fine.

Reply:
Well but this error event won't have that. And you're not going to be able to easily correlate this event with the original `deleteObjectFromProject` event.

Reply:
I may be missing something.  There shouldn't be an original `deleteObjectFromProject` event with which to correlate when deletion fails because it only emits one or the other after the deletion succeeded or failed.

Reply:
Fair - but that still doesn't cover the point that this event as it stands won't have the constructor name property in it.

Reply:
Are we talking about different things?  
The type of object being deleted is showing up as expected in the debugger.

![image](https://user-images.githubusercontent.com/1609827/106336233-526ae480-6243-11eb-9a58-a97fbe1c099a.png)

Reply:
Originally you hadn't included the objectType property in the error event. This seems to be fixed now so all looks good =)